minio/minio
# crawler: Missing folder heal check per set

## Description

Replace the global heal checks with a check that only involves the set that contains the current disk.

This does *not* change the drive-by healing of individual objects found, only deleted objects/folders on the first 2 prefix levels.

## How to test this PR?

* Delete files to observe the performance of the crawler
* Delete backend files in a set. Observe if objects are restored when this disk gets crawled.

## Types of changes
